stratum/hal/lib/common/common.proto

```diff
@@ -1564,6 +1564,14 @@ message IPsecOffloadInfo {
   int32 spi = 1;
 }
 
+// IPsecNotification message is sent from Stratum to gNMI-client based on a
+// target-originated event.
+message VportStateNotification {
+  uint32 global_resource_id = 1;
+  bool state = 2;
```

Comment:
The notification generated from TDI has the state as `uint8`. We should probably keep the same datatype instead of conversion -- this way, it will align with what the user expects from the YANG model